docs/modules/ic-identity-guide/pages/auth-how-to.adoc

```diff
@@ -1,106 +1,106 @@
-= How to use the Internet Identity 
+= How to use Internet Identity
 :keywords: Internet Computer,blockchain,protocol,replica,subnet,data center,canister,developer
 :proglang: Motoko
 :platform: Internet Computer platform
 :IC: Internet Computer
 :company-id: DFINITY
 :sdk-short-name: DFINITY Canister SDK
 
-The Internet Identity enables you to authenticate securely and anonymously when you access applications that use it as an authentication method. A different identity is created for each application you log in to, and you will be able to use all of your registered devices or authentication methods to log in to the same account. This is all accomplished using encrypted smart contracts that run on the {IC} blockchain.
+_Internet Identity_ is a smart contract implemented on the {IC} bockchain. It enables you to authenticate securely and anonymously when you access applications that use it for authentication. An account is associated with an _Identity Anchor_, and a user is free to set up multiple of these. For each application you authenticate to using an Identity Anchor, a different _pseudonym_ specific to that application is created. This pseudonym cannot be linked to the Identity Anchor. You can add multiple _authentication methods_, such as security keys or facial recognition on computers or phones to an Identity Anchor, and you will be able to use all of the added authentication methods to authenticate using that Identity Anchor.
 
-Unlike most authentication methods, your Internet Identity does not require you to set and manage passwords or provide any personal identifying information to applications or to Internet Identity. Instead, you create authentication profiles using a identity anchor and the authentication methods you choose such as facial recognition from a smartphone, your computer unlock password, or a security key. 
+Unlike most authentication methods, Internet Identity does not require you to set and manage passwords or provide any personal identifying information to applications or to Internet Identity.
 
-NOTE: On OSX (including iPhones), authentication using Safari is coupled to your browser profile. If you want to log in to an application in a different browser, or if you use multiple Safari browser profiles, you have to add the combination of your authentication method and the new browser as a new device. See: <<Add a device,`+Add a device+`>>.
+All currently supported authentication methods follow the _WebAuhn_ standard. The following restrictions apply:
 
-Windows users can only use a security key as an authorized device.
+* On OS X (including iPhones), authentication using Safari is coupled to your browser profile. If you want to authenticate to an application in a different browser, or if you use multiple Safari browser profiles, you have to add the combination of your authentication method and the new browser as a new device. See: <<Add a device,`+Add a device+`>>.
```

Comment:
I wish it were that simple... On iOS Face/TouchId work cross-browser while on OSX Chrome and Safari don't interoperate...

Reply:
Oh I see. Thanks for pointing this out. 

I removed `(including iPhones)` and added the sentence: `Note that on iOS, in contrast to OS X, authentication works across browsers.` to the end of the paragraph. This should now be correct, agreed?